library/init/wf.lean

```diff
@@ -106,25 +106,6 @@ section
 end
 end inv_image
 
--- The transitive closure of a well-founded relation is well-founded
-namespace tc
-section
-  parameters {α : Sort u} {r : α → α → Prop}
-  local notation `r⁺` := tc r
-
-  lemma accessible {z : α} (ac : acc r z) : acc (tc r) z :=
```

Comment:
Do you think this lemma should exist in mathlib?

Reply:
https://github.com/leanprover-community/mathlib/pull/14190